.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Report issues with core functionality to Gutenberg
```

Definitely not necessary by any means, but we might consider retaining the pattern of the `name` being a noun and `about` describing the action for consistency with other templates. Please feel free to disregard this suggestion. 
```suggestion
  - name: Bug report (core Gutenberg)
```